cancer_ai/validator/dataset_handlers/image_csv.py

```diff
@@ -40,6 +40,14 @@ async def sync_training_data(self):
                 self.entries.append(ImageEntry(row[0], row[1]))
 
     async def get_training_data(self) -> Tuple[List, List]:
+        """
+        Get the training data.
+
+        This method is responsible for loading the training data and returning it as a tuple of two lists: the first list contains the input data and the second list contains the labels.
```

Comment:
jak piszesz  co zwracasz w returns, to się nie powtarzaj w opisie

Reply:
corrected 😃